web/src/main/client/src/Authenticator.js

```diff
@@ -0,0 +1,52 @@
+const base64 = require('base-64');
+
+let instance = null;
+
+class Authenticator {
+
+  constructor() {
+    if (!instance){
+      instance = this;
+    }
+
+    return instance;
+  }
+
+  authenticate(callback){
+    //Fetches the authentication token from the oauth api
+    fetch('https://happynews-api.svendubbeld.nl/oauth/token', {
+      method: 'post',
+      headers:{
+        "Authorization": "Basic " + base64.encode('happynews-editor:OuNNQtRGBIfUTG2IDICCdOUt'),
```

Comment:
I think we should find a way to move these credentials to a config file on the server instead of in the code.

We can do that in the next iteration though.